### PR TITLE
Add an example to connect to a gps device on com1

### PR DESCRIPTION
**Pull request by tinohager**

### CODE REVIEW

**Comment by tinohager** [comment]
I found the wiki entry also after a long search. I like a quick code example in the readme. You have only a code example of create a new message parser. I think the most people use your project to parse existing messages. In my opinion this should be the first code example.

**Comment by tinohager** [comment]
And it is possible that your code example is outdated?

**Comment by dotMorten** [comment]
This is all great feedback. I should make links to the WIKI more prominent, and yes be better at updating the wiki.
I'm sort if in a transition state currently revamping for v2.0 which will have several breaking changes. I'm a little reluctant to update the doc to this design, since most people are using the current nuget packages and will be looking for doc on how to use those.

The samples should also serve as decent examples on how to use the library though.

File: README.md [hunk 1]
```diff
@@ -22,15 +22,46 @@ Currently supported NMEA messages:
...
+    if (messageType == typeof(NmeaParser.Nmea.Gnss.Gngll))
+    {
+        var gngll = message as NmeaParser.Nmea.Gnss.Gngll;
+        Console.WriteLine($"Gngll - {gngll.Latitude} {gngll.Longitude}");
+    }
+    else if (messageType == typeof(NmeaParser.Nmea.Gnss.Gngga))
+    {
+        var gngga = message as NmeaParser.Nmea.Gnss.Gngga;
+        Console.WriteLine($"Gngga - {gngga.Latitude} {gngga.Longitude}");
+    }
```
[the remark this answers is not in the document]
Comment (tinohager) [reply README.md]: @dotMorten Have you a plan to to choose to v2?

[the remark this answers is not in the document]
Comment (tinohager) [reply README.md]: Maybe you can push a prelease? https://docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages

[the remark this answers is not in the document]
Comment (dotMorten) [reply README.md]: It still needs some work and I've been a bit busy trying to finish some work and conference stuff up. Should have some time in a few weeks to do some proper field testing.
As long as it's pre-release though, I think the same concerns apply that doc should match non-pre-release. It's just an odd transition to v2.0 time right now.

[the remark this answers is not in the document]
Comment (dotMorten) [reply README.md]: ...in hindsight I should probably have done this in a branch for a while #LessonsLearned

[the remark this answers is not in the document]
Comment (tinohager) [reply README.md]: I have switch in my enviorment to v2 and build the dll self. Maybe i send you some updates the next time :)

[the remark this answers is not in the document]
Comment (dotMorten) [reply README.md]: > Have you a plan to switch to .netstandard 2.0?

What would the point be? It's already targeting 1.4

> I have some problems with the UWP Project,

Please be more specific. I'm terrible at guessing problems.

**Pull request closed without merging**